css/styles.css

```diff
@@ -84,3 +89,99 @@ form {
 .go:active {
   transform: scale(0.9);
 }
+
+.background-blue {
+  background-color: #02aff0;
+}
+
+.nav-logo {
+  width: 25%;
+  cursor: pointer;
+  position: relative;
+  right: 0.4rem;
+}
+
+.my-review-nav {
+  text-decoration: none;
+  font-size: 1.5rem;
+  font-weight: 400;
+}
+
+.my-review-nav:hover {
+  color: rgb(221, 226, 232);
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.review-title {
+  background-color: #00678c;
+  padding: 0.3rem 0.5rem;
+  border-radius: 5px;
+  font-size: 1.5rem;
+  font-style: italic;
+  font-weight: 400;
+  margin-bottom: 0.5rem;
+}
+
+.review-sub {
+  background-color: #0091c8;
+  padding: 0.3rem 0.5rem;
+  border-radius: 5px;
+  font-size: 1.5rem;
+  font-weight: 400;
+  margin-bottom: 0.5rem;
+}
```

Comment:
These two rules are identical except for the italic style. You should make this into one rule with all the shared styling, and a separate rule for just italics.